src/main/java/io/confluent/oauth/azure/managedidentity/OAuthBearerLoginCallbackHandler.java

```diff
@@ -43,6 +43,7 @@
 import org.apache.kafka.common.security.oauthbearer.OAuthBearerToken;
 import org.apache.kafka.common.security.oauthbearer.OAuthBearerTokenCallback;
 import org.apache.kafka.common.security.oauthbearer.internals.OAuthBearerClientInitialResponse;
+import org.apache.kafka.common.security.oauthbearer.internals.secured.*;
 import org.apache.kafka.common.security.oauthbearer.secured.*;
```

Comment:
This import can probably now be removed if we change to `<code>OAuthBearerValidatorCallbackHandler</code>` in the following javadocs, removing the `@code`.

Reply:
I was able to remove the `org.apache.kafka.common.security.oauthbearer.secured` but not the `org.apache.kafka.common.security.oauthbearer.internals.secured`